includes/functions/articles.php

```diff
@@ -165,23 +167,36 @@ function get_columns() {
 	 *
 	 * Gets the columns (and sets the internal headers property)
 	 * Gets the \WP_Post for each article ID in the current object's set
-	 *
 	 */
 	function prepare_items() {
 		$columns               = $this->get_columns();
-		$hidden                = [];
-		$sortable              = [];
-		$this->_column_headers = [ $columns, $hidden, $sortable ];
+		$hidden                = array();
+		$sortable              = array();
+		$this->_column_headers = array( $columns, $hidden, $sortable );
 		foreach ( $this->article_ids as $id ) {
 			$post = get_post( $id );
+
+			$post_img_num   = 0;
+			$gallery_images = get_post_galleries( $post, true );
+			if ( ! empty( $gallery_images ) ) {
+				foreach ( $gallery_images as $single ) {
+					$post->post_content .= $single;
+				}
+			}
+			$post_img_num = (int) preg_match_all( '/<img[^>]*>/', $post->post_content, $matches );
+			if ( has_post_thumbnail( $post->ID ) ) {
+				++$post_img_num;
+			}
+			$post->post_img_num = $post_img_num;
+
 			$this->items[] = $post;
 		}
 	}
 
 	/**
 	 * Generates content for a single row of the table
 	 *
-	 * @param object $item The current item
+	 * @param object                                                         $item The current item
```

Comment:
This looks to be an incorrect PHPCS fix because the line below is incorrect (it has the param name first and the type second and those should be switched)

Reply:
@dkotter Yeah, got this one taken care of [here](https://github.com/10up/eight-day-week/pull/120/files#diff-9455c29e5a2f414e80acec3a66761c5e19cf0249ff3de0846acc7ed862adc268R200) since it's unrelated to this issue & [opened a PR](https://github.com/10up/eight-day-week/pull/117#issuecomment-1587677103) for those types of fixes. @peterwilsoncc [commented](https://github.com/10up/eight-day-week/pull/117#issuecomment-1588260721) & mentioned to keep this PR specific to functional changes since I opened that other PR. Let me know if you want me to go ahead and knock it out in this one instead.

Reply:
Updated